{{cookiecutter.project_slug}}/Makefile

```diff
@@ -65,6 +67,7 @@ setup: ### Setup remote environment
 		--env PLATFORMAPI_SERVICE_HOST="." \
 		$(BASE_ENV_NAME) \
 		'sleep 1h'
+	for file in $(PROJECT_PYTHON_FILES); do neuro cp ./$$file $(PROJECT_PATH_STORAGE)/$$file; done
```

Comment:
for making linters work from inside the job, not really necessary for the functionality introduced in htis PR